src/features/user/ManageProjects/components/BasicDetails.tsx

```diff
@@ -180,33 +276,65 @@ export default function BasicDetails({
     reset,
     setValue,
     setError,
+    clearErrors,
   } = useForm({ mode: 'onBlur', defaultValues: defaultBasicDetails });
 
+  const nextStep = () => {
+    handleNext();
+  };
 
   const [acceptDonations, setAcceptDonations] = useState(false);
+  /**
+   * *if project is already had created then user can visit to  other forms using skip button
+   */
+  React.useEffect(() => {
+    if (projectDetails.id) {
+      setSkipButtonVisible(true);
+    }
+  }, [router]);
+
   // const treeCost = watch('treeCost');
 
   React.useEffect(() => {
     if (projectDetails) {
-      const basicDetails = {
-        name: projectDetails.name,
-        slug: projectDetails.slug,
-        classification: projectDetails.classification,
-        countTarget: projectDetails.countTarget,
-        website: projectDetails.website,
-        description: projectDetails.description,
-        acceptDonations: projectDetails.acceptDonations,
-        treeCost: getFormattedNumber(
-          i18n.language,
-          projectDetails.treeCost || 0
-        ),
-        publish: projectDetails.publish,
-        visitorAssistance: projectDetails.visitorAssistance,
-        enablePlantLocations: projectDetails.enablePlantLocations,
-        currency: projectDetails.currency,
-        latitude: projectDetails.geoLatitude,
-        longitude: projectDetails.geoLongitude,
-      };
+      const basicDetails =
+        purpose === 'trees'
+          ? {
+              name: projectDetails.name,
+              slug: projectDetails.slug,
+              classification: projectDetails.classification,
+              countTarget: projectDetails.countTarget,
+              website: projectDetails.website,
+              description: projectDetails.description,
+              acceptDonations: projectDetails.acceptDonations,
+              unitCost: getFormattedNumber(
+                i18n.language,
+                projectDetails.unitCost || 0
+              ),
+
+              visitorAssistance: projectDetails.visitorAssistance,
+              enablePlantLocations: projectDetails.enablePlantLocations,
+              currency: projectDetails.currency,
+              latitude: projectDetails.geoLatitude,
+              longitude: projectDetails.geoLongitude,
+            }
+          : {
+              purpose: projectDetails.purpose,
+              name: projectDetails.name,
+              slug: projectDetails.slug,
+
+              website: projectDetails.website,
+              description: projectDetails.description,
+              acceptDonations: projectDetails.acceptDonations,
+              unitCost: getFormattedNumber(
+                i18n.language,
+                projectDetails.unitCost || 0
+              ),
+              currency: projectDetails.currency,
+              latitude: projectDetails.geoLatitude,
+              longitude: projectDetails.geoLongitude,
+              ecosystems: projectDetails.metadata.ecosystems,
```

Comment:
Selecting Conservation type project breaks the application. It might be breaking here.

![Selecting-conservation-type-project-breaks-the-ui](https://user-images.githubusercontent.com/37466979/167783630-bb6c1364-1787-4516-a0b7-139cbae851d1.PNG)

Reply:
it is fixed